src/algorithm/extremes.rs

```diff
@@ -11,11 +10,6 @@ use types::{Extremes, ExtremePoint};
 // -1, 0. = smallest x
 
 // various tests for vector orientation relative to a direction vector u
-fn up<T>(u: &Point<T>, v: &Point<T>) -> bool
```

Comment:
I'd prefer this to be kept too, in case binary search lands, so just tagging it with `#[allow(dead_code)]` is fine.